### DISCUSSION TITLE
Will there be any v1 releases after 1.13.2

### DISCUSSION THREAD

**Discussion by Kristonitas** [software-mansion/react-native-reanimated#1676] [Q&A]
In my project, there are a few dependancies that depend on reanimated and probably wouldn't work with reanimated 2 and some other basic animations that use reanimated directly. I was wondering if its still a good investment of effort to write code that uses reanimated v1 and if v1 will still be supported for the majority of this year? I'm not talking about new features, but any bugfixes or something like that.

I was looking over documentation, issues and discussions and couldn't find this question being raised.

**Comment by jgonet** [comment]
V1 is going into maintenance mode – we'll only merge bug fixes and releases will be rarer.
